.github/workflows/_reusable_run_tests.yml

```diff
@@ -168,7 +168,7 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ env.PACKAGES_DIR }}
-          key: ${{ runner.os }}-swiftpm-project-${{ hashFiles('**/*.xcworkspace/xcshareddata/swiftpm/Package.resolved') }}
+          key: ${{ runner.os }}-swiftpm-project-${{ hashFiles('**/*.xcworkspace/xcshareddata/swiftpm/Package.resolved') }}-true
```

Comment:
Looks good, however, I'm not sure if it should be `false` instead.

Reply:
I thought the default `ENABLE_DATADOG `value is true, but I think it shouldn't be a problem to use cache with DD.

Reply:
for running tests I would say false also. ENABLE_DATADOG is not defined here